### ISSUE TITLE
Getting ": No such file or directory" when run autolesshint on Ubuntu

### ISSUE THREAD

**Issue by kyrylovoronoi** [automutate/autolesshint#61]
Hi
we have tried install it globally and locally on Ubuntu, but in any case getting
~~~
: No such file or directory
~~~
error
lesshint works perfectly fine in the same time
what have gone wrong?

**Comment by JoshuaKGoldberg** [comment]
Hi kyrylovoronoi! The package isn't yet guaranteed to work on non-Windows systems, as per the notice in the README.md, but I'd like to get that to work. A few clarifying questions for you:
* What full command to run are you using?
* Is this the only output that gets displayed? Any more error logs?
* Are you able to post your source code, or a minimal set of sources that produce the error?
* What exact Ubuntu distribution are you using?

**Comment by giles-v** [comment]
I'm also getting an error much like this on OS X:
```
$ npm run fix:less

> @ fix:less /Users/giles-v/project
> autolesshint public/assets/css/

env: node\r: No such file or directory
```
For regular lesshint:
```
$ npm run lint:less

> @ lint:less /Users/giles-v/project
> lesshint public/assets/css/

Warning: base.less: line 12, col 21, spaceAroundComma: Commas should be followed by one space.
[...]
```

**Comment by danstiner** [comment]
Same problem as https://github.com/JoshuaKGoldberg/tsc-fancy/issues/1, that is when installed `node_modules/autolesshint/bin/autolesshint` has dos line endings that don't work so well on Unix systems.

Just re-publish a new version with a newer version of npm and the problem will go away.

**Comment by JoshuaKGoldberg** [comment]
Thanks @danstiner! Just published version **0.2.0** that should fix this issue.

@kyrylovoronoi @giles-v it's been a while, but please comment here if you happen to try it out again and the issue isn't fixed!
